libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
@MichaelMraka We haven't released it yet, that's what's so special. Everyone here is arguing about bumping from a release that isn't out yet to another release that isn't out yet. There's literally no point in a version bump when we haven't even tagged 0.7.0 for release yet.